tv_apps/dashboard/js/dashboard.js

```diff
@@ -25,25 +26,30 @@
         case 'up':
           if (key === 'down') {
             this.mainSection.dataset.activeDirection = '';
+            this.background.classList.remove('up');
```

Comment:
Let's move `activeDirection` to `body` and get rid of the `background.classList`.